docker/cloudbuild.yaml

```diff
@@ -0,0 +1,52 @@
+options:
+  machineType: "N1_HIGHCPU_32"
+timeout: 1800s
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  entrypoint: 'bash'
+  args: ['-c', 'docker login --username=lutter --password=$$PASSWORD']
+  secretEnv: ['PASSWORD']
+- name: 'gcr.io/cloud-builders/docker'
+  args: ['build', '--target', 'graph-node-build',
+         '--build-arg', 'COMMIT_SHA=$COMMIT_SHA',
+         '--build-arg', 'REPO_NAME=$REPO_NAME',
+         '--build-arg', 'BRANCH_NAME=$BRANCH_NAME',
+         '--build-arg', 'TAG_NAME=$TAG_NAME',
+         '-t', 'gcr.io/$PROJECT_ID/graph-node-build:$SHORT_SHA',
```

Comment:
If we use `docker login`, does that log us into docker hub? Would it then be possible to use `-t graphprotocol/graph-node:$SHORT_SHA` and have it push to the docker hub registry?

Reply:
Yes, and that's how I had it initially, until I realized there's a little more logic around tagged builds that I don't think can be expressed in a Cloud Build file which is why I added `tag.sh`

Reply:
Ok, fair enough